projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+# Datagrid - Style API
+
+## CSS Variables
+
+| CSS Variable Name                                     | Description                                                                  |
+| ----------------------------------------------------- | -----------------------------------------------------------------------------|
+| --clr-datagrid-font-color                             | Changes the text color of datagrid.                                          |
+| --clr-datagrid-default-border-color                   | Changes the border color of datagrid when default.                           |
+| --clr-datagrid-icon-color                             | Changes the header icons color of datagrid.                                  |
+| --clr-datagrid-row-hover-color                        | Changes the row background color of the data table when hovers over element. |
+| --clr-datagrid-row-hover-font-color                   | Changes the row text color of data table when hovers over element.           |
+| --clr-datagrid-action-toggle-color                    | Changes the toggle action button color.                                      |
+| --clr-datagrid-pagination-btn-color                   | Changes the table bottom pagination buttons color.                           |
+| --clr-datagrid-pagination-btn-disabled-color          | Changes the table bottom pagination buttons color when disabled.             |
```

Comment:
Pls remove all "table" and "bottom" occurrences in the file.
We have only one pagination, so it's not necessary to specify that it's bottom. Also it's just "datagrid". Not "table" or "table datagrid".

Reply:
Removed all table and bottom occurrences in the file.